core/src/main/scala/com/rackspace/com/papi/components/checker/ByteArrayServletInputStream.scala

```diff
@@ -13,14 +13,24 @@
  *   See the License for the specific language governing permissions and
  *   limitations under the License.
  */
-package com.rackspace.com.papi.components.checker.servlet
+package com.rackspace.com.papi.components.checker
```

Comment:
Got it.  The validator-base one is for testing. This should still like in com.rackspace.com.papi.components.checker.servlet  not useful outside of that context at all.  You can have the tests use the com.rackspace.com.papi.components.checker.servlet one.